py/server/tests/test_udf_args.py

```diff
@@ -287,7 +287,7 @@ def f6(*args: np.int32, col2: np.ndarray[np.int32]) -> bool:
 
         with self.assertRaises(DHError) as cm:
             t1 = t.update(["X1 = f6(X, Y=null)"])
-        self.assertIn("not compatible with annotation", str(cm.exception))
+        self.assertIn("f6: Expected argument (col2) to be one of [class [I], got boolean", str(cm.exception))
```

Comment:
A user, especially a python user is going to have a hard time with an error with type strings like `[I]`.  Can the error be made more user accessible?

Reply:
I think we can but it won't be a straightforward/non-trivial effort. Filed https://github.com/deephaven/deephaven-core/issues/5279